### ISSUE TITLE
Change persistence system to use Realm or something not filesystem-based

### ISSUE THREAD

**Issue by pwightman** [ZuseApp/zuse-ios#83]
We had lots of problems of saving things to disk, then it not being there when reloading the collection view. We should use a proper persistent store instead of just the filesystem.

**Comment by ghost** [comment]
I completely agree.  I wouldn't feel comfortable releasing v1 without having a better way to save and load files.  Most of our crashes happen because of race conditions with the filesystem.
